tests/Regress/Mmap.hsc

```diff
@@ -60,7 +60,7 @@ mprotect :: Ptr a -> CSize -> CInt -> IO ()
 mprotect addr len prot =
     throwErrnoIfMinus1_ "mprotect" $ c_mprotect addr len prot
 
-foreign import ccall unsafe "sys/mman.h mmap"
+foreign import ccall unsafe "hashable_mmap"
```

Comment:
Why can't we use the `capi` FFI calling-convention which was invented exactly for this use case to avoid having to write such trivial wrappers as in `cbits/mmap.c`?